Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import argparse
+import glob
+import logging
+import os
+import o3de.export_project as exp
+import pathlib
+import platform
+import shutil
+import sys
+import time
+import psutil
+
+from o3de.export_project import process_command
+from o3de import manifest
+from typing import List
+
+
+# This script is meant to be run via o3de cli export-project command, not from Python directly!
+# This export script operates in a project-centric manner, meaning all build artifacts are centered around the project folder. 
+# In order to use this script, the command should look like the following (on windows use o3de.bat, on linux use o3de.sh):
+
+# cd path\to\o3de\engine
+# .\scripts\o3de.bat  export-project --export-scripts path\to\o3de\ExportScripts\export_standalone_monolithic_project_centric.py \
+# --project-path path\to\project --log-level INFO --output-path path\to\output --build-non-mono-tools --config release --archive-output zip \
+# --seedlist path\to\seedlist1.seed --seedlist path\to\seedlist2.seed <etc..> 
```

Comment:
Alternatively the current working directory can be in the project folder and the path to the engine can be specified.

```python
# Example invocation on Linux
'''
$ cd <project-path>
$ <engine-path>/scripts/o3de.sh export-project --export-scripts ExportScripts/export_standalone_monolithic.py \
 --project-path . --log-level INFO --output-path <install-path> --build-non-mono-tools --config release --archive-output xz \
 --seedlist ./Assets/seedlist1.seed --seedlist ./Assets/seedlist2.seed <etc..> 
'''
```